### PR TITLE
Generate WKT

### PR DESCRIPTION
**Pull request by dio**
This produces:

```
$ make gen
$ tree website/generated/
website/generated/
├── booking
│   └── v1
│       └── doc.json
└── google
    └── protobuf
        ├── any.proto.json
        ├── api.proto.json
        ├── compiler
        │   └── plugin.proto.json
        ├── descriptor.proto.json
        ├── duration.proto.json
        ├── empty.proto.json
        ├── field_mask.proto.json
        ├── source_context.proto.json
        ├── struct.proto.json
        ├── timestamp.proto.json
        ├── type.proto.json
        └── wrappers.proto.json
```

Signed-off-by: Dhi Aurrahman <dio@rockybars.com>

### CODE REVIEW

**Comment by dio** [comment]
For googleapis it is easier since we can use `buf`.

**Comment by dio** [comment]
There are still missing title and description. I'll fix the generator on that.

**Comment by imballinst** [comment]
Oooo, nice! Thanks for working on this. I'll take a look on the generated JSONs tomorrow (but feel free to merge when you feel like it).

File: Makefile [hunk 1]
```diff
@@ -74,8 +74,16 @@ help: ## Describe how to use each target
 gen: $(BUF_V1_MODULE_DATA) $(yarn) ## Generate files from proto files
 	@printf "$(ansi_format_dark)" $@ "generating files..."
 	@$(buf) generate
+	@$(MAKE) gen-wkt
 	@printf "$(ansi_format_bright)" $@ "ok"
 
+generated_dir := website/generated
+gen-wkt: $(protoc) $(protoc-gen-doc)
```
Comment (imballinst) [inline Makefile:81]: Just for my curiosity: what does WKT stand for again?

Comment (dio) [reply Makefile:81]: Well-known types https://buf.build/beta/wellknowntypes

**Pull request merged**